pixi.toml

```diff
@@ -0,0 +1,120 @@
+[project]
+name = "lumen"
+channels = ["pyviz/label/dev", "bokeh", "conda-forge"]
+platforms = ["linux-64", "osx-arm64", "osx-64", "win-64"]
+
+[tasks]
+# download-data = 'python scripts/download_data.py'
+install = 'python -m pip install --no-deps --disable-pip-version-check -e .'
+
+[activation.env]
+PYTHONIOENCODING = "utf-8"
+
+[environments]
+test-39 = ["py39", "test-core", "test"]
+test-310 = ["py310", "test-core", "test"]
+test-311 = ["py311", "test-core", "test"]
+test-312 = ["py312", "test-core", "test"]
+test-core = ["py312", "test-core"]
+docs = ["py311", "doc"]
+build = ["py311", "build"]
+lint = ["py311", "lint"]
+
+[dependencies]
+bokeh = "*"
+holoviews = ">=1.17.0"
+hvplot = "*"
+intake = "<2"
+jinja2 = ">3.0"
+numpy = "*"
+packaging = "*"
+pandas = "*"
+panel = ">=1.3.0"
+param = ">=1.9.0"
+pip = "*"
+
+[feature.py39.dependencies]
+python = "3.9.*"
+
+[feature.py310.dependencies]
+python = "3.10.*"
+
+[feature.py311.dependencies]
+python = "3.11.*"
+
+[feature.py312.dependencies]
+python = "3.12.*"
+
+# =============================================
+# =================== TESTS ===================
+# =============================================
+[feature.test-core.dependencies]
+psutil = "*"
+pytest = "*"
+pytest-cov = "*"
+pytest-github-actions-annotate-failures = "*"
+pytest-rerunfailures = "*"
+pytest-xdist = "*"
+
+[feature.test-core.tasks] # So it is not showing up in the test-ui environment
+test-unit = 'pytest lumen/tests -n logical --dist loadgroup'
+
+[feature.test.dependencies]
+dask-core = "*"
+fastparquet = "*"
+matplotlib-base = ">=3.4" # Ubuntu + Python 3.9 installs old version matplotlib (3.3.2)
+msgpack-python = "*"
+toolz = "*"
+# sql
+intake-sql = "*"
+python-duckdb = "*"
+sqlalchemy = "*"
+
+# [feature.ai.dependencies]
+# datashader = "*"
+# python-duckdb = "*"
+# instructor = ">=1"
+# nbformat = "*"
+# openai = "*"
+# pyarrow = "*"
+# pydantic = ">=2.8.0"
+#
+# [feature.ai-local.dependencies]
+# huggingface_hub = "*"
+#
+# [feature.ai-llama.dependencies]
+# llama-cpp-python = "*"
```

Comment:
I don't really know what to do with these. As far as I can see, they were never used in the CI. 

They are still available as optional dependencies.